aurora/contracts/src/AuroraErc20FastBridge.sol

```diff
@@ -43,6 +43,7 @@ contract AuroraErc20FastBridge is Initializable, UUPSUpgradeable, AccessControlU
     NEAR public near;
     string public fastBridgeAccountIdOnNear;
     string public auroraEngineAccountIdOnNear;
+    string public nativeAuroraTokenAccountIdOnNear;
```

Comment:
This corrupt the storage, please move it to the last storage slot.

Reply:
fixed: https://github.com/aurora-is-near/fast-bridge-protocol/pull/113/commits/7dee54bb8a323b0f1fe63d818bf2e305e5e2607f